ts/output/chtml.ts

```diff
@@ -152,31 +158,62 @@ CommonOutputJax<N, T, D, CHTMLWrapper<N, T, D>, CHTMLWrapperFactory<N, T, D>, CH
    * @override
    */
   public styleSheet(html: MathDocument<N, T, D>) {
-    if (this.chtmlStyles && !this.options.adaptiveCSS) {
+    if (this.chtmlStyles) {
+      if (this.options.adaptiveCSS) {
+        //
+        // Update the style sheet rules
+        //
+        const styles = new CssStyles();
+        this.addWrapperStyles(styles);
+        this.updateFontStyles(styles);
+        this.adaptor.insertRules(this.chtmlStyles, styles.getStyleRules());
+      }
       return this.chtmlStyles;  // stylesheet is already added to the document
     }
     const sheet = this.chtmlStyles = super.styleSheet(html);
```

Comment:
Is there a reason why this can't be done on `this.chtmlStyles` directly?

Reply:
Sorry, I'm not sure I understand the question.  If you are asking about why the intermediate `styles` object in lines 166 to 169, the reason is that the information that comes from the Wrapper classes and font are just JSON objects, not actual style definition strings, as needed for `insertRules()`.  The `CssStyles()` object is the intermediary that can convert the JSON to the appropriate CSS strings.